docs/how_jstz_works.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 👨‍⚖️ How jstz works?
+---
+
+<script setup>
+import VPButton from "vitepress/dist/client/theme-default/components/VPButton.vue";
+</script>
+
```

Comment:
Following our discussion yesterday, this doc will be moving to our internal docs (likely a section in `CONTRIBUTING.md`). So these vitepress headers / buttons are no longer required